Remove ShARC from transferring and choosing resources (#1952)
rcgsheffield/sheffield_hpc: hpc/Choosing-appropriate-resources.rst

* the **"wallclock"** (the time your job took if measured by a clock on your wall)
* the consumed **"CPU time"** (a number of seconds of compute, derived directly from the amount of CPU time used on all cores of a job).

How to determine these values can be seen below using the :ref:`seff` command as below:

.. tabs::

> 

Reviewer: Do we need separate tabs for Stanage vs Bessemer?

If your job / program supports multiple cores, you need to assess whether it supports SMP (symmetric multiprocessing) where you can only use CPUs on 1 node or MPI (message passing interface) where you can access as many nodes, CPUs and cores as are available.

> For SMP only type parallel processing jobs: you can use a maximum of 64 cores on Stanage and 40 cores on Bessemer. 

Reviewer: ```suggestion
For single-node jobs: you can use a maximum of 64 cores on Stanage and 40 cores on Bessemer. 
```

> For multiple node MPI type parallel processing jobs: these can run on Stanage and although you can access as many cores as are available you must consider how long a job will take to queue waiting for resources compared the the decrease in time for the job to complete computation.

Reviewer: ```suggestion
For multiple-node MPI-type parallel processing jobs: these can run on Stanage and although you can access as many cores as are available you must consider how long a job will take to queue waiting for resources compared the the decrease in time for the job to complete computation.
```

> Single node MPI type parallel jobs can run on Stanage and Bessemer.

Reviewer: ```suggestion
Single-node MPI-type parallel jobs can run on Stanage and Bessemer.
```

----------------------------------

**By using the emailing parameters of the sbatch command:**

Submit your job with ``sbatch`` by specifying very generous memory and time requirements to ensure that it runs to completion" and also using the ``-M`` and ``-m abe`` or  ``--mail-user=`` and ``--mail-type=ALL``   parameters to receive an email-report. The mail message will list the maximum memory usage ( maxvmem / MaxVMSize  ) as well as the wallclock time used by the job.

> .. tabs::

Reviewer: Do we need tabs here now ShARC has gone?
